packages/terra-status/src/terra-dev-site/doc/status/Status.1.doc.mdx

```diff
@@ -0,0 +1,73 @@
+import { Badge } from 'terra-status/package.json?dev-site-package';
+
+import StatusDefault from '../example/StatusDefault?dev-site-example';
+import StatusImage from '../example/StatusImage?dev-site-example';
+import StatusIcon from '../example/StatusIcon?dev-site-example';
+import StatusArrange from '../example/StatusArrange?dev-site-example';
+
+import StatusPropsTable from 'terra-status/src/Status?dev-site-props-table';
+
+<Badge />
+
+# Terra Status
+
+The status component provides a customizable color indictor to signify a specific condition.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-status`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
+
+```jsx
+import Status from 'terra-status';
+```
+
+```css
```

Comment:
Can you create a section for to add come context to this css example? Maybe name it `Adding custom class name` and describe how  is the syntax that is expected. By itself, this css don't make sense.

Reply:
done here https://github.com/cerner/terra-core/pull/2873/commits/88bf523f4a82084da9f5a28b1039deacb7d482c6